models/Proposal.php

```diff
@@ -6,7 +6,9 @@
 
 class Proposal extends databaseModels\Proposal
 {
+    public $count_reviews = null;
```

Comment:
- Should be camelCase
- Where do you use it ?

Reply:
No it couldn't be camelCase because it's used in query to find the approved proposals. If your model doesn't have the attribute declared somewhere it can't display the active data provider.